.github/templates/linux_ci_workflow.yml.in

```diff
@@ -178,3 +178,86 @@ jobs:
           docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
           # Prune all of the docker images
           docker system prune -af
+  {% if enable_doc_jobs %}
+  pytorch_python_doc_build:
+    runs-on: ubuntu-18.04
```

Comment:
This needs to run on a `linux.2xlarge` in order to have access to ECR

Reply:
I see, updated